randovania/data/json_data/prime3.txt

```diff
@@ -1246,24 +1429,32 @@ Asset id: 14006603351279167667
 
 > Pickup (Energy Cell); Heals? False
   * Pickup 34; Major Location? False
+  > Dock to Burrow
+      After Hidden Court Burrow Gate
   > Top of Halfpipe
       Trivial
 
 > Top of Halfpipe; Heals? False
   > Pickup (Missile Expansion)
       Trivial
   > Pickup (Energy Cell)
-      Ship Grapple
+      Ship Grapple and After North Jungle Court Battery
   > Bottom of Halfpipe
       Trivial
 
 > Bottom Floor; Heals? False
   > Dock to Ruined Shrine
-      Trivial
+      After Hidden Court Golem
+  > Dock to Burrow
+      All of the following:
+          After Hidden Court Burrow Gate
+          Grapple Swing or Use Screw Attack (Space Jump)
   > Dock to Falls of Fire
       Trivial
   > Bottom of Halfpipe
-      Use Screw Attack (Space Jump)
+      After Hidden Court Golem and Use Screw Attack (Space Jump)
```

Comment:
Change this to connect to Hidden Court Hall instead, with the requirements:
Any of the Following:
- "After Golem" and Grapple Swing
- Screw Attack